internal/services/custom_rules/service.go

```diff
@@ -32,30 +32,30 @@ import (
 // load custom rules from an configuration file.
 type Service struct {
 	config      *config.Config
-	customRules map[languages.Language][]engine.Rule
+	customRules map[languages.Language][]text.Rule
 }
 
 // NewCustomRulesService create a new custom rule service with all rules
 // loaded from configuration file path.
 func NewCustomRulesService(cfg *config.Config) *Service {
 	service := &Service{
 		config: cfg,
-		customRules: map[languages.Language][]engine.Rule{
-			languages.CSharp:     make([]engine.Rule, 0),
-			languages.Dart:       make([]engine.Rule, 0),
-			languages.Java:       make([]engine.Rule, 0),
-			languages.Kotlin:     make([]engine.Rule, 0),
-			languages.Yaml:       make([]engine.Rule, 0),
-			languages.Leaks:      make([]engine.Rule, 0),
-			languages.Javascript: make([]engine.Rule, 0),
-			languages.Nginx:      make([]engine.Rule, 0),
+		customRules: map[languages.Language][]text.Rule{
+			languages.CSharp:     make([]text.Rule, 0),
+			languages.Dart:       make([]text.Rule, 0),
+			languages.Java:       make([]text.Rule, 0),
+			languages.Kotlin:     make([]text.Rule, 0),
+			languages.Yaml:       make([]text.Rule, 0),
+			languages.Leaks:      make([]text.Rule, 0),
+			languages.Javascript: make([]text.Rule, 0),
+			languages.Nginx:      make([]text.Rule, 0),
 		},
 	}
 	return service.loadCustomRules()
 }
 
 // Load implements formatters.CustomRules
-func (s *Service) Load(lang languages.Language) []engine.Rule {
+func (s *Service) Load(lang languages.Language) []text.Rule {
```

Comment:
Why not use the interface `engine.Rule` instead? Since we declare this rule interface on engine package, I think that we should use this interface on every **possible** place instead of `text.Rule`.